jx_module/modules/registry/main.tf

```diff
@@ -0,0 +1,21 @@
+resource "azurerm_container_registry" "acr" {
```

Comment:
I'd suggest referring to the existing modules (such as ACR, networking, etc )from other active projects from Microsoft github as Cobalt https://github.com/microsoft/cobalt/tree/master/infra/modules/providers/azure/container-registry

Reply:
Not sure I agree with increasing dependencies on other projects where there isn't a lot of value there (i.e. these modules are not complex in nature). The modules in this repo are quite thin wrappers on ARM resources and are just so we can  decompose how we use these services within the context of jx, i.e. optionally enable ACR module or not.

Reply:
The returned values:
- this module was already tested and unit/integration tests using terratest already there ([PR](https://github.com/microsoft/cobalt/pull/407) is about to be merged)
- I can assure you that we have addressed all special cases and best practices while developing these modules. 

In case you don't want to add more dependencies, I can suggest to fork(copy the modules.

Reply:
We still need to test here as we need to validate that the cluster service principals can push/pull from the registry. Suggest we merge this PR as the work has already been done and if we want to offload this to a cobalt  project that we open a subsequent PR to do so. Just this is holding a lot of work up not getting this merged is all :)

Reply:
I can feel the pain by having a PR opened for weeks ... Let's submit an issue to discuss this next meeting please and will approve it for now.

Reply:
;)